samza-core/src/main/java/org/apache/samza/zk/ZkJobCoordinator.java

```diff
@@ -64,21 +65,25 @@
   private String newJobModelVersion;  // version published in ZK (by the leader)
```

Comment:
What is the purpose of this variable? Last time I checked, it was not being used anywhere.